milestone2.ts

```diff
@@ -79,11 +79,15 @@ const userData: Volunteer[] = [
 // Questions Start Here
 
 // Question 1: Lambda Functions
-/* Define a new lambda function that finds the average age of the people in the data. 
+/* Define a new lambda function that finds the average age of the users in the data. 
   Hint: user data is stored in the userData object above. 
 */
 type GetNumber = (data: Volunteer[]) => number;
-let findAverage: GetNumber; // Define lambda function here
+let findAverage: GetNumber = (data) => {
+  const get_ages = data.filter(datas => datas.age) // get all ages
```

Comment:
This doesn't do anything at the moment, if you wanted an array of ages only, you would use `map`, not `filter`